docs/Writing_PRs.md

```diff
@@ -15,7 +15,9 @@ want to know what it concerns.
 Describe what the PR accomplishes at a high-level, including anything that would
 help give context to the reader:
 
-* Links to the fogbugz ticket, trello card, error log, or spec doc
+* External links - JIRA tickets, trello cards, error logs, or spec docs are helpful, but 
+  don't assume future readers of the PR will have access to those links. Consider reiterating information 
+  from an external link in the description if especially useful.
```

Comment:
maybe "...but do keep in mind that internal information, like specific project information, does not belong in PRs"?

Reply:
Good point. See dc68c6d199394d05fae0fd0929014824cd1623b4